pkg/stackit_api_manager/client/.openapi-generator/FILES

```diff
@@ -4,20 +4,24 @@ client.go
 configuration.go
 docs/APIManagerServiceApi.md
 docs/GoogleProtobufAny.md
-docs/Metadata.md
```

Comment:
this file in theory should be deleted.
but i can't see that it is deleted on this PR.
For this I recommend deleting the whole `client` folder and run `make generate-client` again.